src/java.base/share/classes/java/lang/ScopedValue.java

```diff
@@ -611,10 +606,11 @@ private Object findBinding() {
      * Returns the value of this scoped value if bound in the current thread, otherwise
      * returns {@code other}.
      *
-     * @param other the value to return if not bound, can be {@code null}
+     * @param other the value to return if not bound
      * @return the value of the scoped value if bound, otherwise {@code other}
      */
     public T orElse(T other) {
+        Objects.requireNonNull(other);
```

Comment:
Shouldn't the NPE be specified in the Javadoc?

Reply:
In the class specification, right above the API notes:
```
 * <p> Unless otherwise specified, passing a {@code null} argument to a method in this
 * class will cause a {@link NullPointerException} to be thrown.
```